frontend/src/components/Pages/MissionDefinitionPage/MissionDefinitionPage.tsx

```diff
@@ -48,21 +62,29 @@ const StyledMissionDefinitionPage = styled.div`
     gap: 1rem;
     margin: 2rem;
 `
+const StyledButton = styled(Button)`
+    width: 150px;
+`
+const StyledInspectionFrequencyDiv = styled.div`
+    > * {
+        padding: 10px;
+    }
+`
 
-interface IProps {
-    missionDefinition: CondensedMissionDefinition
-    updateMissionDefinition: (missionDefinition: CondensedMissionDefinition) => void
-}
-
-function KeyValuePairDisplay({ left, right }: { left: string; right: any }) {
+function KeyValuePairDisplay({ left, right, onEdit }: { left: string; right: any; onEdit?: () => void }) {
     return (
-        <>
+        <StyledFormItem>
             <Typography
                 variant="body_long_bold"
                 group="paragraph"
                 color={tokens.colors.text.static_icons__secondary.rgba}
             >
                 {left}
+                {onEdit && (
+                    <Button style={{ padding: '5px' }} variant="ghost" onClick={onEdit}>
+                        <Icon name={Icons.Edit} size={16} />
+                    </Button>
+                )}
```

Comment:
The ones with button becomes taller and doesn't align
![image](https://github.com/equinor/flotilla/assets/33518988/976350b9-fae3-4366-a20b-e3540cb62b83)